docs/variants/msi_z690/faq.md

```diff
@@ -0,0 +1,89 @@
+# FAQ
+
+## Will this work on my `MSI PRO Z690-A WIFI DDR5` or `MSI PRO Z690-A DDR5` ?
+
+Not yet. The build is configured for the DDR4 only. It might be extended with
+the DDR5 variant in the future. **Do not try that**.
+
+## Will this work on my `MSI PRO Z690-A DDR4` ?
+
+Although we do not own that one and cannot give promises, we believe that it
+should work the same as the `MSI PRO Z690-A WIFI DDR4`. This is almost the
+same board, just with no WiFi card.
```

Comment:
@macpijan can we include that board into testing process if someone will contribute 2-3 setups?

Reply:
You mean: if someone donates 2-3 setups for us for testing? I believe so. Even one should probably be enough.

Reply:
@macpijan so here we should think about "how you can help" page, partially sent in previous newsletter, but what we see here, someone can donate hardware DRAM modules, CPU or mainabords.

Reply:
We should inform people how to do that.